content/07.pathogenesis.md

```diff
@@ -1,8 +1,16 @@
 ## Pathogenesis
 
-### Mechanism of Host Infection by SARS-CoV-2
-
-This section would also be great for the introduction of zoonotic diseases which has been shown to be the origin of SARS-CoV2.
+Pathogenesis is made up of three major components: entry, replication, and spread [@isbn:0-9631172-1-1 {chap. 45}].
```

Comment:
At least the bits about viral entry are probably best dealt with in that section.

Reply:
I think this is addressed [in changes here](https://github.com/greenelab/covid19-review/pull/620/commits/9f20d4d135f6d7083d83976158ef15644a7cdd84)